src/config/option/display_option.rs

```diff
@@ -19,6 +19,7 @@ pub struct DisplayOption {
     pub _show_preview: bool,
     pub _sort_options: SortOption,
     pub _tilde_in_titlebar: bool,
+    pub _line_nums: u8,
```

Comment:
You can make a separate enum for this instead of using arbitrary int values.
Something like:
```rust
pub enum LineNumberStyle {
    None,
    Relative,
    Absolute,
}
```
And the `no` option can be `none`